Makefile

```
@@ -115,6 +131,7 @@ run-backend: ## run the backend server (development mode)
 	@$(COMPOSE) up -d backend
 	@echo "Waiting for backend to be up and running..."
 	@$(COMPOSE_RUN) dockerize -wait tcp://$(ES_COMPOSE_SERVICE):$(ES_PORT) -timeout 60s
+	@$(COMPOSE_RUN) dockerize -wait http://ralph:secret@$(RALPH_COMPOSE_SERVICE):$(RALPH_RUNSERVER_PORT)/whoami -timeout 60s
```

Why not using [health checks](https://github.com/openfun/ralph/blob/master/src/ralph/api/routers/health.py#L29) instead?

```suggestion
	@$(COMPOSE_RUN) dockerize -wait http://ralph:secret@$(RALPH_COMPOSE_SERVICE):$(RALPH_RUNSERVER_PORT)/__heartbeat__ -timeout 60s
```